epictrack-web/src/components/shared/filterSelect/type.ts

```diff
@@ -24,6 +24,7 @@ declare module "react-select/dist/declarations/src/Select" {
       getOptionLabel?: (option: any) => string;
       getOptionValue?: (option: any) => string;
       label?: string;
+      maxWidth?: string;

```

Comment:
Do we have to define and pass these props now explicitly? Is the example in the ControlledTextField too hard to achieve here?

![image](https://github.com/bcgov/EPIC.track/assets/91914654/01ffdbc2-b345-4362-b5cc-da10ffa297e9)

Reply:
how do we pass that into the style prop of the select field its the maxwidth of the component not the character length